ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -71,26 +72,28 @@
       </:body>
     </form.fieldset>
 
+    <Rose::Separator />
+
     <form.fieldset>
       <:title>{{t 'resources.host-catalog.form.provider-details'}}</:title>
       <:body>
         <form.radioGroup
-          @name='aws-plugins'
+          @name='aws_plugins'
```

Comment:
The line below (106) uses `aws_plugin` rather than `aws_plugins`. I am wondering the reason why we use plural here and singular below.

Reply:
I'm still working on cleaning up the names a bit to be consistent.  more updates on this soon!

Reply:
Awesome!! thanks for clarification